content/blog/kotlin-meet-grpc.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Kotlin, meet gRPC
+date: 2020-04-16
+authors:
+- name: Louis Wasserman
+  position: Google
+- name: Brent Shaffer
+  position: Google
```

Comment:
@lowasser, @bshaffer: I can add links to, say, your GitHub user profiles. Would you like that, or might you want me to  link to something else?

Reply:
Also let me know if you'd like a detailed `position` entries.

Reply:
Actually, I just pushed a commit with some updates: adding the full title, and links to your GH page. PTAL